src/v2/Apps/Search/Components/ZeroState.tsx

```diff
@@ -22,7 +22,9 @@ export const ZeroState: FC<ZeroStateProps> = props => {
     >
       <Box m={3} textAlign="center">
         <Text variant="subtitle" mb={1}>
-          {hasFilters ? "No results found." : `No results found for "${term}".`}
+          {hasFilters
+            ? "No results found."
+            : `No results found for \u201C${term}\u201D.`}
```

Comment:
**suggestion (non-blocking)**: This totally works, but note that these can also be inlined as the actual glyph with `option-[` and `option-shift-[` (or  good ol' `ctrl-cmd-space`). Which imo makes them less cryptic/brittle.

Reply:
I cannot ever remember these for some reason (though i have em/en dashes burned into my brain), and always go to https://rdquo.net/ to copypasta

Reply:
I actually prefer using something that stands out as being very obviously different than ". If you feel strongly I can make this change, otherwise I feel like it's a horse a piece.